src/searchbar.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+    import Typeahead from './Typeahead.vue'
+    export default {
+      extends: Typeahead,
+      computed: {
+        primitiveData() {
+          if (this.data) {
+            var matches = []
+            var words
+            this.data.map(value => {
+              var query = new RegExp(this.value, 'i')
+              var hasHeadingMatch = false
+              var title = value.title
```

Comment:
Since we have gotten rid of the `.toLowerCase()`, the assignment might no longer be necessary, just use `value.title` below. E.g.:

```js
matches.push({'title': value.title,
```

Same thing for `keywords` and `headings`.